packages/test/test-end-to-end-tests/src/test/summarizeIncrementally.spec.ts

```diff
@@ -0,0 +1,245 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
Am a bit unclear on the logical divide of test cases between the other file and this new file. The file names also look similar. Can you explain this better?

Reply:
The other file is not aptly named IMO. That one is about incremental summary of DDS's content. By default, the FF runtime does incremental summary of the entire data store or the entire DDS. However, a DDS can choose to also do it for its content and the implementation can vary from DDS to DDS.

But, I think they can move to a single file. Let me do that.

Reply:
One second thoughts, I think its better for them to be in separate files because the other tests have a lot of setup code which will reduce the readability of this one (which is much simpler). I renamed the other file.

Reply:
Thanks. this is a better categorization of tests. One is about incremental summary of DDS content and the other is about entire DataStore/DDS. Thanks